frontend/src/components/pomodoroview/PomodoroView.jsx

```diff
@@ -69,7 +69,53 @@ function RemainingTimeContextProvider(props) {
   const [isClockPlaying, setIsClockPlaying] = useState(false);
   const [workDurationMinutes, setWorkDurationMinutes] = useState(25);
   const [breakDurationMinutes, setBreakDurationMinutes] = useState(5);
+  const [minute, setMinute] = useState(25);
+  const [second, setSecond] = useState(0);
 
+  useEffect ( () => {
+    loadItemFromLocalStorage("startTime", setStartTime, "string")
+    loadItemFromLocalStorage("isUserBreaking", setIsUserBreaking, "boolean")
+    loadItemFromLocalStorage("ClockKey", setClockKey, "number")
+    loadItemFromLocalStorage("isClockPlaying", setIsClockPlaying, "boolean")
+    loadItemFromLocalStorage("workDurationMinutes", setWorkDurationMinutes, "number")
+    loadItemFromLocalStorage("breakDurationMinutes", setBreakDurationMinutes, "number")
+    loadItemFromLocalStorage("Minute", setMinute, "number")
+    loadItemFromLocalStorage("Second", setSecond, "number")
+    loadItemFromLocalStorage("Second", setSecond, "number") 
```

Comment:
Duplication

Reply:
Thanks. Fixed. 
One con of local storage is that it can only store string. The item has to be converted to its original type while loading. Another option is to stringify and parse the item during storing and loading.